packages/synapse-react-client/src/components/CardDeck/CardDeck.Desktop.tsx

```diff
@@ -60,51 +61,47 @@ export default function CardDeckDesktop({
             {' '}
             {title}{' '}
           </span>
-          {/* {link && (
-            <p className="Programs__Card__header__info__link">
-              <a
-                className="highlight-link"
-                href={link}
-                target="_blank"
-                rel="noreferrer"
-              >
-                Visit website
-              </a>
-            </p>
-          )} */}
         </div>
       </div>
-      <div className="CardDeck__Card__summary">
+      <div
+        className={classNames(
+          `CardDeck__Card__summary`,
+          cardDeckType && `CardDeck__Card__summary--${cardDeckType}`,
+        )}
+      >
         <p> {description} </p>
-        <LargeButton
-          color="secondary"
-          variant={cardDeckType === 'cckp' ? 'outlined' : 'contained'} // Update variant based on cardDeckType
-          href={ctaButtonURL}
-          sx={{
-            marginLeft: 0,
-            fontSize: '14px',
-            boxShadow: 'none',
-            color: 'primary.main',
-            '&:hover': {
-              backgroundColor: 'primary.main',
-              color: 'white',
-            },
-            ...(cardDeckType === 'b2ai' && {
-              width: '200px',
-              backgroundColor: 'white',
-            }),
-            ...(cardDeckType === 'cckp' && {
-              width: '170px',
-              borderColor: 'primary.main',
+
+        {cardDeckType === 'b2ai' ? (
+          <Link href={ctaButtonURL} underline="none">
+            {ctaButtonText}
+          </Link>
+        ) : (
+          <LargeButton
+            color="secondary"
```

Comment:
Can we just make the top-level color primary and then remove the color styles?

```suggestion
            color="primary"
```